src/workers/postUpdated.ts

```diff
@@ -255,10 +255,25 @@ const updatePost = async ({
   content_type = PostType.Article,
 }: UpdatePostProps) => {
   const postType = contentTypeFromPostType[content_type];
-  const databasePost = await entityManager
+  let databasePost = await entityManager
     .getRepository(postType)
     .findOneBy({ id });
 
+  // If we don't find the post, we need to check if it's a youtube video and
+  // try to find it again as an article
+  if (!databasePost && content_type === PostType.VideoYouTube) {
+    await entityManager
+      .createQueryBuilder()
+      .update(Post)
+      .set({ type: PostType.VideoYouTube })
```

Comment:
Why don't we set it to every content type which is not article?

Reply:
Allow it for all posts, not just YouTube?
Was thinking to just do it for YouTube for now, to not introduce any unintended side-effects (if there are any).

```ts
if (!databasePost) {
  await entityManager
    .createQueryBuilder()
    .update(Post)
    .set({ type: content_type })
    .where('id = :id', { id })
    .execute();

  databasePost = await entityManager
    .getRepository(postType)
    .findOneBy({ id });
}
```

Reply:
Yeah think this is a good proposal, simply because we might introduce more types soon.
even know type might be article, but render as collection or whatever we might want to update it.

Reply:
yes

Reply:
Updated